jimp/index.d.ts

```diff
@@ -0,0 +1,163 @@
+// Type definitions for jimp 0.2
+// Project: https://github.com/oliver-moran/jimp#readme
+// Definitions by: Jack Works <https://github.com/Jack-Works>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
```

Comment:
Per the guideline, can you use either 4 spaces or all tabs for indentation instead? Thanks